src/Quartz/Util/StringKeyDirtyFlagMap.cs

```diff
@@ -208,8 +208,16 @@ public virtual void Put(string key, Guid value)
         /// </summary>
         public virtual int GetInt(string key)
         {
-            var obj = this[key];
+            if (!this.ContainsKey(key))
```

Comment:
to avoid doing the lookup twice, should use `!this.TryGetValue(key, out var obj)`